OpenSim/Tests/Wrapping/testWrapCylinder.cpp

```diff
@@ -57,6 +69,51 @@ namespace {
             "PathSegment{start: " << path.start << ", end: " << path.end << "}";
     }
 
+    PathSegment operator*(
+        const SimTK::Rotation& rot,
+        const PathSegment& path)
+    {
+        return PathSegment{
+            rot * path.start,
+            rot * path.end,
+        };
+    }
+
```

Comment:
Add `enum class Direction { Positive, Negative };` and replace `bool` flags with that enum, so that downstream code can become `Direction windingDirection = Direction::Positive;` rather than `bool positiveDirection` (etc.)

(just a suggestion from a common code-smell guideline of "be careful about `bool` flag arguments to functions)

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#i4-make-interfaces-precisely-and-strongly-typed

Reply:
oh, nice link :)